src/cursor.rs

```diff
@@ -27,7 +27,8 @@ use {Client, CommandType, Error, ErrorCode, Result, ThreadedClient};
 use apm::{CommandStarted, CommandResult, EventRunner};
 
 use bson::{self, Bson};
-use common::{ReadMode, ReadPreference};
+use common::{ReadMode, merge_options, ReadPreference};
```

Comment:
Oops, I actually meant to put it alphabetically, which would be at the beginning. Is that okay too?